docs/source/matplotlib/intro.rst

```diff
@@ -38,6 +42,9 @@ previous example, we used the
 :meth:`~cartopy.mpl.geoaxes.GeoAxes.coastlines` method
 to add coastlines to the map.
 
+To save the figure, use matplotlib's :func:`~matplotlib.pyplot.savefig`
```

Comment:
Matplotlib

Reply:
@QuLogic You mean I need to capitalize it, right? I'll assume you're not just stating your appreciation for the library...